sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Models/OperationsLiveTests.cs

```diff
@@ -106,6 +107,7 @@ public async Task TrainingOperationCanPollFromNewObject()
         }
 
         [Test]
+        [Ignore("CHECK REGIONS")]
         public async Task CopyModelOperationCanPollFromNewObject()
```

Comment:
Check regions?

Reply:
Changos!!! this was a note for me to check this test again. Thank you!!!

Reply:
Enabled again